### DISCUSSION TITLE
how to compile mpv with my own changes?

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#15348] [Q&A] [answered]
Hi, I have no experience with compiling anything but I need to make my own mpv version, because I have to patch something out because of this issue https://github.com/mpv-player/mpv/issues/15340
Can someone help me with that?
I'm on Windows 10

Thank you :)

**Comment by kasper93** [comment, marked as answer]
https://github.com/mpv-player/mpv/blob/master/DOCS/compile-windows.md

**Reply by geextahslex** [reply]
Okay I guess this isn't straightforward ^^'

**Comment by guidocella** [comment]
That patch is for libplacebo so you would have to compile that too.

**Reply by geextahslex** [reply]
Are you sure? Isn't this patch in the mpv files? 
https://github.com/mpv-player/mpv/issues/15340#issuecomment-2489829554

**Reply by guidocella** [reply]
That's not a patch, it's just copy pasted code. I was talking about https://github.com/mpv-player/mpv/issues/12707#issuecomment-1774077987
